gems/pending/Gemfile

```diff
@@ -1,56 +1,58 @@
-raise "Ruby versions less than 2.2 are unsupported!" if RUBY_VERSION < "2.2.0"
-
```

Comment:
Where did this go?  Please don't remove this.  It prevents accidental usage of the wrong Ruby version, which is important for devs that need to switch back and forth to old version.